projects/nutanix-cloud-native/cloud-provider-nutanix/ATTRIBUTION.txt

```diff
@@ -50,10 +50,10 @@ https://github.com/modern-go/concurrent
 ** github.com/modern-go/reflect2; version v1.0.2 --
```

Comment:
@adiantum Can you confirm that these are the only changes to the attribution file? On main we went from `v0.4.0` to `v0.4.1`, here we're going from `v0.3.2` to `v0.4.1`.

Reply:
Yeah, sorry, I didn't noted that make attribution checksum failed when I run it... I'll recreate PR.

Reply:
I think you can probably just copy over the attribution file from main, if the Git tag and Go version are the same. And if you have wrong checksums, the presubmit validation will fail but it will give you the expected correct checksum values, which you can use to update the PR and make the validation happy.